changelog/65214.fixed.md

```diff
@@ -0,0 +1 @@
+Stopped erasing previously collected code coverage on test failure reruns
```

Comment:
I don't think this requires a changelog. can you remove?

Reply:
Done